adoc/admin-kubernetes-changes.adoc

```diff
@@ -0,0 +1,74 @@
+== From Kubernetes 1.17 to 1.18
+
+This documentation page lists all the behavioral changes and API deprecations that will happen from Kubernetes 1.17 to 1.18. This is going to be relevant when migrating to {productname} 4.3.x, since it will bring Kubernetes 1.18.
+
+=== Architectural changes in {productname}
+
+`hyperkube` has been removed from the Kubernetes project and is now community maintained. That being said, its future is uncertain, and it was no longer really needed for {productname}. Thus, we have decided to remove this binary starting from {productname} 4.3.0 and we are now providing binaries and container images for each Kubernetes component.
+
+This should not affect you directly: it will be transparent. That being said, if you relied on `hyperkube` specifically for some reason, take this into account.
+
+=== API deprecations
+
+The following APIs have been deprecated:
+
+- All resources under *apps/v1beta1* and *apps/v1beta2*: from now on use *apps/v1* instead.
+- *daemonsets*, *deployments*, *replicasets* under *extensions/v1beta1*: from now on use *apps/v1* instead.
+- *networkpolicies* under *extensions/v1beta1*: from now on use *networking.k8s.io/v1* instead.
+- *podsecuritypolicies* resources under *extensions/v1beta1*: from now on use *policy/v1beta1* instead.
+
+=== Behavioral changes
+
+In this section we will highlight some relevant changes that might interest you for the new Kubernetes version.
+
+==== Core
+
+- link:https://github.com/kubernetes/enhancements/issues/853[#853] Configurable scale velocity for HPA.
+- link:https://github.com/kubernetes/enhancements/issues/1393[#1393] Provide OIDC discovery for service account token issuer.
+- link:https://github.com/kubernetes/enhancements/issues/1513[#1513] CertificateSigningRequest API.
+
+==== Scheduling
+
+- link:https://github.com/kubernetes/enhancements/issues/1451[#1451] Run multiple Scheduling Profiles [Alpha]
+- link:https://github.com/kubernetes/enhancements/issues/895[#895] Even pod spreading across failure domains [Beta]
+- link:https://github.com/kubernetes/enhancements/issues/1258[#1258] Add a configurable default Even Pod Spreading rule [Alpha]
```

Comment:
should we mention alpha things?

Reply:
I'd say so, as to give customers a heads up for incoming features for future versions.